easybuild/easyconfigs/p/PythonPlus/PythonPlus-3.7.4-foss-2018b-v20.02.1.eb

```diff
@@ -7,15 +7,17 @@ versionsuffix = '-v20.02.1'   # In format YY.MM.IncrementedReleaseNumber.
 homepage = 'https://www.python.org/'
 description = """The PythonPlus bundle contains a plain vanilla Python supplemented with a selection of add-on modules."""
 
-#
-# numpy requires BLAS -> use full foss toolchain as opposed to GCCcore.
 # scipy requires compiler option -fPIC.
-#
 toolchain = {'name': 'foss', 'version': '2018b'}
 toolchainopts = {'pic': True}
 
+exts_defaultclass = 'PythonPackage'
+# overwrite default sanity check
+exts_filter = ('python -c "import scipy"', "")
+
+# depend on numpy because of https://github.com/biocore/scikit-bio/issues/1734
 dependencies = [
-    ('Python', version, '-bare'),
+    ('numpy', '1.17.0', '-Python-3.7.4'),
```

Comment:
Can you use version veriable here as opposed to hardcoded 3.7.4?

Reply:
Possibly. I didn't use it because the version of the PythonPlus module means something different then the version of Python (even though it has the same value).